### PR TITLE
Fix #2694 Link to the graduation reports in the blog.

### LINKED ISSUE #2694: UX to link to graduation reports in blog
## Currently

Graduation reports [live on the site](https://testpilot.firefox.com/experiments/no-more-404s), but we'd like to move them to the blog.

## Proposal

For experiments like [Page Shot](https://testpilot.firefox.com/experiments/page-shot) we'd  like to be able to keep the majority of page content, but provide a CTA to visit a link on the Test Pilot Medium Channel highlighting the Graduation report.

## Acceptance

- [ ] It should provide a CTA to the previous experiment graduation report
- [ ] It should still be possible to view the original experiment page content

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: frontend/src/app/components/LocalizedHtml.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ export default class LocalizedHtml extends Localized {
                 }
               );
             }
-            throw new Error('ftl string did not have as many anchors as the jsx');
+            throw new Error(`ftl string "${this.props.id}" did not have as many anchors as the jsx`);
```
Comment (fzzzy) [inline frontend/src/app/components/LocalizedHtml.js:126]: This is unrelated, but it helped me debug something while working on this branch, might as well get it into master.

**Comment by fzzzy** [comment]
Still need to:

- [x] Add strings for localization.
- [x] Add stories.
- [x] Add tests.
- [x] Resolve conflicts after https://github.com/mozilla/testpilot/pull/2859 lands.

**Comment by fzzzy** [comment]
The test failures seem to be a temporary problem with circle.

**Comment by fzzzy** [comment]
The temporary circle problem obscured the lint errors.

File: locales/en-US/app.ftl [hunk 2]
```diff
@@ -260,3 +260,10 @@ warningMissingPrefTitle=Developing Test Pilot?
 warningMissingPrefDetail=When running Test Pilot locally or in development environments, special configuration is required. Please see <a>our documentation</a> for details.
 warningBadHostnameTitle=Unapproved hostname!
 warningBadHostnameDetail=The Test Pilot site may only be accessed from testpilot.firefox.com, testpilot.stage.mozaws.net, testpilot.dev.mozaws.net, or example.com:8000. Please see <a>our documentation</a> for details.
+
+// Text to show to users when an experiment has graduated
+[[graduated]]
+graduationReportButton=Graduation Report
+graduationNoticeTitle=This experiment has ended
+graduationNoticeReportNotReady=We are working on a full report. Check back soon for the details.
```
Comment (flodolo) [inline locales/en-US/app.ftl]: Why are we duplicating two existing strings? Are those used anywhere?

```
experimentGradReportPendingTitle = This experiment has ended
experimentGradReportPendingCopy = We are working on a full report. Check back soon for the details.
```

Comment (fzzzy) [reply locales/en-US/app.ftl]: Thank you for pointing this out! I searched for "graduation" but there were no hits, and "grad" hit a bunch of other things that were not related. I will re-use the existing strings.

File: frontend/src/app/components/GraduatedNotice/index.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
+
+
+
+.graduated-notice {
```
Comment (meandavejustice) [inline frontend/src/app/components/GraduatedNotice/index.scss]: nit, extra new lines

File: frontend/src/app/components/GraduatedNotice/stories.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+
+import React from 'react';
```
Comment (meandavejustice) [inline frontend/src/app/components/GraduatedNotice/stories.js]: nit, new lines at beginning of file

File: frontend/src/app/components/GraduatedNotice/stories.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+import { storiesOf } from '@storybook/react';
+
+import GraduatedNotice from './index';
```
Comment (meandavejustice) [inline frontend/src/app/components/GraduatedNotice/stories.js]: inconsistent spacing of new lines

**Comment by fzzzy** [comment]
Since this has a string change, this shouldn't land until the new sprint starts.

**Comment by lmorchard** [comment]
New sprint is go, and we updated staging. Merging!

**Pull request merged**